orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/basic/main.tf

```diff
@@ -57,6 +57,13 @@ module "orc8r-app" {
   secretsmanager_orc8r_name = module.orc8r.secretsmanager_secret_name
   seed_certs_dir            = "~/secrets/certs"
 
+  managed_certs_create      = false
+  managed_certs_enabled     = false
+  nms_managed_certs_enabled = false
+
+  managed_certs_route53_access_key = ""
+  managed_certs_route53_secret_key = ""
```

Comment:
Is there a way to generate this programmatically? Seems like a pain to have to manually inject this into the main.tf

Reply:
In the official method, we have to provide the keys so cert-manager can add TXT records to Route53 whenever the certificate is about to expire.
https://cert-manager.io/docs/configuration/acme/dns01/route53/

Reply:
I am looking for a way if we can add these values dynamically from a file.

Reply:
I am not considering security at this point so for now they can just use the keys they have for Orc8r deployment.

Reply:
Yeah I figure there should be a way to hook this up automatically with Terraform... lmk what you find though

Reply:
Hi @hcgatewood, I found a way to generate a new set of secret keys for Cert-manager to use. This way we won't have to add the keys manually. But this will conflict if we create multiple Orc8r in the same AWS account due to the same IAM username. Do you want me to add some random values at the end to resolve this issue?
https://github.com/ShubhamTatvamasi/terraform-aws-iam-access-key/blob/master/main.tf

Reply:
So I have just updated the code to generate `random_id ` so we don't get any conflicts.
Now integrating this is our cert-manager terraform module.